vspace/src/main/webapp/resources/bootstrap/css/bootstrap.css

```diff
@@ -1082,7 +1085,7 @@ html {
   -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
 }
 body {
-  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-family: "Roboto", Helvetica, Arial, sans-serif;
```

Comment:
you shouldn't need this anymore since you've added the font for all tags/class above